ctest-next/tests/basic.rs

```diff
@@ -37,7 +34,7 @@ fn bless_equal(new_file: impl AsRef<Path>, old_file: impl AsRef<Path>) {
     let new_content = fs::read_to_string(&new_file).unwrap().replace("\r", "");
     let old_content = fs::read_to_string(&old_file).unwrap().replace("\r", "");
 
-    let equal = new_content != old_content;
+    let equal = new_content == old_content;
     if env::var("LIBC_BLESS").is_ok() && !equal {
         fs::write(old_file, &new_content).unwrap();
     } else {
```

Comment:
You probably don't actually need to check `equal` before writing, no harm in overwriting the file if they're equal. So you can check `LIBC_BLESS` at the beginning and write the file if so, then read the file and assert it's equal.